SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from spectral_cube import SpectralCube\n",
+    "import spectral_cube\n",
+    "spectral_cube.__version__\n",
+    "spectral_cube.__path__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(75, 250, 250) and unit=Jy / beam:\n",
+       " n_x:    250  type_x: RA---SIN  unit_x: deg    range:   266.534072 deg:  266.554577 deg\n",
+       " n_y:    250  type_y: DEC--SIN  unit_y: deg    range:   -28.713958 deg:  -28.695975 deg\n",
+       " n_s:     75  type_s: VRAD      unit_s: km / s  range:      -43.509 km / s:     104.685 km / s"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube1vel = cube1.with_spectral_unit(u.km/u.s, velocity_convention='radio', rest_value=139.483699*u.GHz)\n",
+    "cube1vel"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(478, 420, 420) and unit=Jy / beam:\n",
+       " n_x:    420  type_x: RA---SIN  unit_x: deg    range:   266.537002 deg:  266.551600 deg\n",
+       " n_y:    420  type_y: DEC--SIN  unit_y: deg    range:   -28.711371 deg:  -28.698569 deg\n",
+       " n_s:    478  type_s: VRAD      unit_s: km / s  range:     -118.278 km / s:     203.359 km / s"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube2vel = cube2.with_spectral_unit(u.km/u.s, velocity_convention='radio', rest_value=217.104984*u.GHz)\n",
+    "cube2vel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the shape of the cube, we can see the H2CS cube is narrower in velocity, so we'll use that as the target spectral reprojection (otherwise, we would have empty velocity channels on either side of the smaller cube that would be getting reprojected).  However, the SiO cube is the smaller footprint on the sky."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Do some quicklooks of the peak intensity to see what we expect to see:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube.max at 0x2ad7b51dd9d8>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n",
+      "/blue/adamginsburg/abulatek/spectral-cube/spectral_cube/spectral_cube.py:443: RuntimeWarning: All-NaN slice encountered\n",
+      "  **kwargs)\n"
+     ]
+    }
+   ],
+   "source": [
+    "cube1.max(axis=0).quicklook()"
```

Comment:
On one hand, I use this convenience all the time because it's convenient... and I'd like people to see how to do that... but on the other, I agree, and we should just do what you said.

Reply:
I agree that it's clearer to unpack this! Added in the edited version.